### PR TITLE
Move keyboard shortcuts table to traceviewer-react-components

### PR DESCRIPTION
**Pull request by hriday-panchasara**
This would allow it to be used by both theia and vscode extension

### CODE REVIEW

**Review by bhufmann — DISMISSED** [review]
Looks good to me. Thanks!

**Comment by hoangphamEclipse** [comment]
Hi @hriday-panchasara, there have been some changes made regarding the CSS classes in the trace extension. Any CSS variables that take up different values in different environment (e.g Theia vs VsCode) must be made mappable. That is, the variable prefix must be `--trace-viewer`. You can see what exactly that you need to do in the following PR #977.

File: packages/react-components/style/trace-explorer.css [hunk 1]
```diff
@@ -315,17 +315,34 @@
     font-size: var(--trace-viewer-ui-font-size1);
     min-height: 23px;
     white-space: nowrap;
+    font-weight: bold;
 }
 
-.shortcuts-table i {
-    color: var(--trace-viewer-button-background);
+.shortcuts-icon {
+    color: var(--theia-button-background);
```
Comment (hoangphamEclipse) [inline packages/react-components/style/trace-explorer.css:322]: Please rebase the code. CSS variables that start with `--theia` should be replaced by `--trace-viewer`.

File: packages/react-components/style/trace-explorer.css [hunk 2]
```diff
@@ -315,17 +315,34 @@
...
+}
+
+.shortcuts-table-keybinding-key {
+    display: inline-block;
+    border-radius: 3px;
+    vertical-align: middle;
+    font-size: 11px;
+    padding: 3px 5px;
+    margin: 0 2px;
+    background-color: var(--theia-keybindingLabel-background);
```
Comment (hoangphamEclipse) [inline packages/react-components/style/trace-explorer.css:341]: CSS variables that start with --theia should be replaced by --trace-viewer. Then, these variables can be mapped to Theia variables in trace-viewer.css. See PR #977.

**Review by hoangphamEclipse — CHANGES_REQUESTED** [review]

**Comment by hoangphamEclipse** [comment]
Thank you. LGTM.

**Review by hoangphamEclipse — APPROVED** [review]

**Review by bhufmann — APPROVED** [review]

**Pull request merged**